src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,97 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize: $pf-global--FontSize--sm;
+
+// label
+$pf-chart-global-label--FontSize: $pf-chart-global--FontSize;
+$pf-chart-global-label--Padding: 10px;
+
+// Layout Props
+$pf-chart-global-layout--Padding: 50px;
+$pf-chart-global-layout--Height: 300px;
+$pf-chart-global-layout--Width: 450px;
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area-stroke--Width: 2px;
+
+// Axis Chart
+$pf-chart-axis-axis-stroke--Width: 1px;
+$pf-chart-axis-axis-label--Padding: 100px;
+$pf-chart-axis--ticks--Width: 1px;  // this should be a var for size so not sure if width is appropriate
+
+// Bar Chart
+$pf-chart-bar--Width: 10px;
+
+// Box Plot Chart
+$pf-chart-boxplot-max--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-max-stroke--Width: 1px;
+$pf-chart-boxplot-median--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-median-stroke--Width: 1px;
+$pf-chart-boxplot-min--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-min-stroke--Width: 1px;
+$pf-chart-boxplot-q1--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-q3--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-box--Width: 20px;
+
+// Bullet Chart
+$pf-chart-bullet--comparative-measure-line-warning--BorderColor: $pf-color-orange-300;
```

Comment:
Yes, please. 

We will have a separate theme files for bullet chart, donut, and the new donut utilization charts below. However, we still need the vars to use with the components.

The props for the donut chart can be found here:
https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-charts/src/components/ChartTheme/themes/theme-donut.js

The props for the donut utilization chart are here:
https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-charts/src/components/ChartTheme/themes/theme-donut-utilization.js

The props for the new donut utilization chart with static thresholds are here:
https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-charts/src/components/ChartTheme/themes/theme-donut-threshold.js

Reply:
Updated based on these files. Some questions. 
- In pie chart, what is padAngle?
- Those font sizes should be 14 and 24 instead of 16 and 26, so they relate to the global font sizes 
- Do you need a variable for: orientation: vertical?

- inside of the donut key value pairs, is "pie". Do those variables relate to pie or donut? I've seen the same height/width of 230 be used under pie but in the pie variation and in the pie inside of donut.

Reply:
padAngle is the space between the pie (i.e., dount) chart segments. This is an example with padAngle=1, which looks like 3 pixels in the design.

![Screen Shot 2019-05-30 at 11 48 36 AM](https://user-images.githubusercontent.com/17481322/58645243-f82cba00-82d0-11e9-9ee0-2a4ef7b4fce3.png)

Reply:
pie and donut are separate components, but Victory only supports pie charts. When creating a donut theme, we merge the donut properties on top of the pie properties. That gives us the look of the donut, but it's actually a pie chart.

Reply:
The legend component has a default orientation, so probably not necessary?

Reply:
That makes sense about pie and donut, and since only colors are defined for donut it keeps the properties of the pie separate.

Also I see in the material.js theme that the legend has an orientation of 'vertical' which is different to the victory file where the legend has an orientation of 'horizontal', so i think ill keep it in there for now?